src/mon/MgrMap.h

```diff
@@ -36,17 +38,21 @@ class StandbyInfo
 
   void encode(bufferlist& bl) const
   {
-    ENCODE_START(1, 1, bl);
+    ENCODE_START(2, 1, bl);
```

Comment:
Unrelated... the 1 and 2 here are the version for this struct only (StandbyInfo)